packages/ui-signer/src/Queue.tsx

```diff
@@ -98,18 +99,18 @@ export default class Queue extends React.Component<Props, State> {
     return id;
   }
 
-  queueExtrinsic = ({ extrinsic, accountNonce = new BN(0), accountId }: QueueTx$Extrinsic): QueueTx$Id => {
+  queueExtrinsic = ({ accountId, accountNonce, extrinsic }: PartialQueueTx$Extrinsic): QueueTx$Id => {
     return this.queueAdd({
-      accountNonce,
-      extrinsic,
-      accountId
+      accountId,
+      accountNonce: accountNonce || new BN(0),
+      extrinsic
     });
   }
 
-  queueRpc = ({ accountNonce = new BN(0), accountId, rpc, values }: QueueTx$Rpc): QueueTx$Id => {
+  queueRpc = ({ accountId, accountNonce, rpc, values }: PartialQueueTx$Rpc): QueueTx$Id => {
     return this.queueAdd({
-      accountNonce,
       accountId,
+      accountNonce: accountNonce || new BN(0),
```

Comment:
The actual fix (same thing that is done on line 105) - the rest are just plumbing to get the types correct